src/app/metadata/features/dataset-expansion-panel/dataset-expansion-panel.component.ts

```diff
@@ -34,8 +34,21 @@ export class DatasetExpansionPanelComponent {
   #metadata = inject(MetadataService);
 
   hit = input.required<Hit>();
+  hitContent = computed(() => this.hit().content);
 
   summary = this.#metadata.datasetSummary;
+  studiesSummary = computed(() => this.summary().studies_summary);
+  studiesSummaryStats = computed(() => this.summary().studies_summary.stats);
+  samplesSummary = computed(() => this.summary().samples_summary);
+  samplesSex = computed(() => this.summary().samples_summary.stats.sex);
```

Comment:
```suggestion
  samplesSex = computed(() => this.samplesSummaryStats().sex);
```

similarly in other places